flow/core/params.py

```diff
@@ -242,7 +247,8 @@ def add(self,
             num_vehicles=0,
             car_following_params=None,
             lane_change_params=None,
-            color=None):
+            color=None,
+            energy_model = None):
```

Comment:
minor: i would put `energy_model` above `color` since it is kind of the same "level" as the upper args.